awx/ui_next/src/components/DetailList/Detail.jsx

```diff
@@ -25,24 +25,29 @@ const DetailValue = styled(({ fullWidth, ...props }) => (
   `}
 `;
 
-const Detail = ({ label, value, fullWidth, className }) => {
+const Detail = ({ label, value, fullWidth, className, dataCy = false }) => {
   if (!value && typeof value !== 'number') {
     return null;
   }
 
+  let labelCy = dataCy ? `${dataCy}-label` : null
+  let valueCy = dataCy ? `${dataCy}-value` : null
+
   return (
     <>
       <DetailName
         className={className}
         component={TextListItemVariants.dt}
         fullWidth={fullWidth}
+        data-cy={labelCy}
```

Comment:
I did things with :oden: case and :camel: case in this way so that React wouldn't yell at me about custom components. Is this right?

Reply:
`data-cy` is kebabed :+1: